### PR TITLE
Add Contributing Guidelines and Code of Conduct 

### PR DESCRIPTION
**Pull request by Yuvraj-kadale**
This PR is a part of issue #737. I have drafted a general one please let me know if any changes are required. 
Thanks! @ensonic

### CODE REVIEW

File: Code_Of_Conduct.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Project maintainers have the right to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct, and will communicate reasons for moderation decisions when appropriate. 
+
+## Scope
+
+This Code of Conduct applies to project administrators, mentors and all contributors within all community spaces.
+
+## Enforcement
+
+Instances of harassment, abuse or any unacceptable behavior may be reported to project administrators responsible for enforcement by emailing (  ). All complaints will be reviewed and investigated promptly and fairly.
```
Comment (ensonic) [inline Code_Of_Conduct.md:46]: @bjost2s any suggestions what mail-address to use here?

File: Code_Of_Conduct.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+**Consequence**: A clearly written, private warning from project administrators providing clarity around the nature of the violation and an explanation of why the behavior was inappropriate. An apology may be requested. 
+
+### 2. Warning
+
+**Community Impact**: A violation through a single incident or series of actions. 
+
+**Consequence**: A warning with consequences for continued behavior. No interaction with the people involved, including unsolicited interaction with those enforcing the Code of Conduct, for a specified period of time. This includes avoiding interaction in all community spaces as well. Violating these terms may lead to a temporary or permanent ban. 
+
+### 4. Permanent Ban
```
Comment (ensonic) [inline Code_Of_Conduct.md:66]: Please renumber to "3."

**Comment by ensonic** [comment]
Thanks a lot. Looks good to me.

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- Work on the issue(s) assigned to you. 
+- Add all the files/folders needed.
+- After you've made changes or made your contribution to the project add changes to the branch you've just created by:
+```
+# To add all new files to branch Branch_Name
+
+$ git add .
+```
+## Step 5 : Commit
+- To commit give a descriptive message for convenience
```
Comment (ensonic) [inline Contributing.md:70]: Please add "A good commit message focuses on the why instead of the what.".

File: Code_Of_Conduct.md (hunk 1 above)
Comment (ensonic) [reply Code_Of_Conduct.md:46]: @bjost2s friendly ping

Comment (bjost2s) [reply Code_Of_Conduct.md:46]: I think we should use our general support email adress which is support-o-r@iais.fraunhofer.de
@ensonic , @Yuvraj-kadale , sorry for the delay

**Comment by ensonic** [comment]
Thanks!

**Pull request merged**